src/das_cli.py

```diff
@@ -25,11 +20,12 @@ def das_cli(ctx):
 
 
 das_cli.add_command(config)
-das_cli.add_command(server)
+das_cli.add_command(db)
 das_cli.add_command(faas)
 das_cli.add_command(metta)
 das_cli.add_command(example)
 das_cli.add_command(logs)
+das_cli.add_command(jupyter_notebook)
```

Comment:
Shouldn't it be `jupyter-notebook` to follow proper command line variable/command name convention?

Reply:
This is just the function name, the command name will be `jupyter-notebook`